docs/building_blocks_for_flexible_content.md

```diff
@@ -519,6 +519,16 @@ blocks:
 
 In the example above only the text in the govspeak and header blocks will be searchable.
 
+### Utility blocks
```

Comment:
Looks like you've gone to the trouble of Title Casing every other heading, but missed this one.

Reply:
Ah lol, it's because the CI complains about Title Casing for headers describing actual blocks, but doesn't care about these intermediate headers. I'll fix it though.

Reply:
Or.. hmm, will I? I think other L1/L2/L3 headings are not title-cased. Maybe I should leave it for the moment.

Reply:
Fine by me!